thyrel-web/src/components/Home/AvatarCard.tsx

```diff
@@ -0,0 +1,63 @@
+import { Avatar, Icon } from 'rsuite';
+import { baseColor } from '../../styles/colors';
+import Box from '../../styles/Box';
+// import styled from '@emotion/styled';
+import { css } from '@emotion/css';
+
+type AvatarCardProps = {
+  image?: string;
+  size?: 'md' | 'lg';
+  onShuffle?: () => void;
+};
+
+// const StyledAvatar = styled(Avatar)({
+//   width: '48px',
+//   height: '48px',
+// });
+
+export default function AvatarCard({
+  //children,
+  image,
+  size,
+  onShuffle,
+}: AvatarCardProps) {
+  console.log(image);
+  return (
+    <Box
+      alignItems="center"
+      width={256}
+      height={256}
+      className={css({
+        position: 'relative',
+      })}>
+      <button
+        onClick={() => onShuffle?.()}
+        className={css({
+          backgroundColor: baseColor,
+          outline: 'none',
+          borderRadius: '50%',
+          position: 'absolute',
+          left: '0',
+          bottom: '0',
+          width: '48px',
+          height: '48px',
+          boxShadow: '0px 6px 1px rgba(30, 70, 70, 0.55)',
+          '&:active': {
+            bottom: -6,
+            boxShadow: 'none',
+          },
+        })}>
+        <Icon icon="random" />
+      </button>
```

Comment:
Ici y'a beaucoup de style.
Tu peux faire ceci (hors de ton component) : 
```js
const ShuffleButton = styled.button({
  // .... tous les styles que tu as mis ici
})

```
puis tu peux utiliser ton composant comme ça  :
```suggestion
      <ShuffleButton
        onClick={() => onShuffle?.()}>
        <Icon icon="random" />
      </ShuffleButton>
```